libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -221,8 +221,8 @@ class AP_Motors {
     float               _lateral_in;                // last lateral input from set_lateral caller
     float               _throttle_avg_max;          // last throttle input from set_throttle_avg_max
     LowPassFilterFloat  _throttle_filter;           // throttle input filter
-    spool_up_down_desired _spool_desired;           // desired spool state
-    spool_up_down_mode  _spool_mode;                // current spool mode
+    DesiredSpoolState _spool_desired;           // desired spool state
+    SpoolState  _spool_state;                // current spool mode
```

Comment:
small indentation issue?  maybe it's just the web viewer..

Reply:
Fixed.